### DISCUSSION TITLE
Is there a way to make pressing enter attempt to go to the next step?

### DISCUSSION THREAD

**Discussion by Twoscore23** [vueform/vueform#324] [Questions] [answered]
Currently when I press enter it either does nothing if not on a input element or it goes to the previous step. Is there anyway to make it attempt the next step with or without being on an input element?

**Comment by adamberecz** [comment, marked as answer]
Here's a possible way to achieve that:
https://stackblitz.com/edit/github-jttfwt?file=src%2FApp.vue

**Reply by Twoscore23** [reply]
Awesome that worked thanks
